docker-compose.yml

```diff
@@ -0,0 +1,110 @@
+version: '3'
+services:
+
+  #PHP Service
+  app:
+    build:
+      context: laravel
+      dockerfile: Dockerfile
+      args:
+        USERNAME: $DB_USERNAME
+    image: digitalocean.com/php
+    container_name: app
+    restart: unless-stopped
+    tty: true
+    working_dir: /var/www
+    env_file:
+      - ${PWD}/.env
+    volumes:
+      - ${PWD}/laravel:/var/www
+      - ${PWD}/.env:/var/www/.env
+      - vendor:/var/www/vendor
+      - node_modules:/var/www/node_modules
+    networks:
+      - app-network
+    depends_on:
+      - db
+
+  #NodeJS - uncomment for websockets
+  sockets:
+    build:
+      context: node
+      dockerfile: Dockerfile
+    container_name: sockets
+    ports:
+      - "3000:3000"
+    networks:
+      - app-network
+    depends_on:
+      - app
+
+  #Nginx Service
+  webserver:
+    image: ghcr.io/linuxserver/swag
+    container_name: swag
+    restart: unless-stopped
+    environment:
+      PUID: 1000
+      PGID: 1000
+      TZ: $TIMEZONE
+      URL: $SITE_URL
+      SUBDOMAINS: $SUBDOMAINS
+      VALIDATION: http
+      ONLY_SUBDOMAINS: "true"
```

Comment:
Does this mean using the volunteer database without a subdomain is not supported?